src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int FIGURE_ARRAY_SIZE = 6;
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] arrayOfFigure = new Figure[FIGURE_ARRAY_SIZE];
+        for (int i = 0; i < arrayOfFigure.length; i++) {
+            arrayOfFigure[i] = i < arrayOfFigure.length / 2
+                    ? figureSupplier.getRandomFigure()
+                    : figureSupplier.getDefaultFigure();
+        }
```

Comment:
Seems that you do not display anything 
<img width="774" alt="Screenshot 2022-12-12 at 12 53 03" src="https://user-images.githubusercontent.com/29378916/207027564-6b8c6150-edaa-4e26-86d2-6aa625f1aa4e.png">